invenio_administration/assets/semantic-ui/js/invenio_administration/src/actions/ActionModal.js

```diff
@@ -33,11 +34,11 @@ ActionModal.propTypes = {
   children: PropTypes.object,
   modalOpen: PropTypes.bool,
   resource: PropTypes.object.isRequired,
+  appName: PropTypes.string,
```

Comment:
IMHO `appName` should be required to avoid having different admin views that can be potentially overriden by the same component, if this varabiable is missing. That's why I raise an error (see `base.py`). I will do the change unless someone has a different opinion